src/app/batch_generate_keys/batch_generate_keys.ml

```diff
@@ -0,0 +1,92 @@
+open Core
+open Signature_lib
+
+let gen_keys count =
+  Quickcheck.random_value ~seed:`Nondeterministic
+    (Quickcheck.Generator.list_with_length count Public_key.Compressed.gen)
+
+let output_keys =
+  let open Command.Let_syntax in
+  Command.basic ~summary:"Generate the given number of public keys on stdout"
+    (let%map_open count =
+       flag "--count" ~aliases:["count"] ~doc:"NUM Number of keys to generate"
+         (required int)
+     in
+     fun () ->
+       List.iter (gen_keys count) ~f:(fun pk ->
+           Format.printf "%s@." (Public_key.Compressed.to_base58_check pk) ))
+
+let output_cmds =
+  let open Command.Let_syntax in
+  Command.basic ~summary:"Generate the given number of public keys on stdout"
```

Comment:
@mrmr1993  is this summary accurate?  shouldn't it be more like "Generate and send the given number of transactions" ? there are a few other places where the provided documentation string doesn't seem to match what it actually does, unless I'm misunderstanding something